index.js

```diff
@@ -21,6 +21,7 @@ module.exports.CORE_SCHEMA         = require('./lib/schema/core');
 module.exports.DEFAULT_SCHEMA      = require('./lib/schema/default');
 module.exports.load                = loader.load;
 module.exports.loadAll             = loader.loadAll;
+module.exports.loadMultiYaml       = loader.loadMultiYaml;
```

Comment:
Thank you for the pull request. However the new `loadMultiYaml` seem to do exactly what `loadAll` function above does, and it's not immediately clear why we need both.

Reply:
You're correct, I misread the documentation. I think a better idea would be to showcase the example of outputting the multi-document sources. E.g. the case where we don't apply the iterator.

I was under the belief the loadAll() function was only loading them in one at time(not storing to an array directly).